experimental/dds/tree/src/EditLog.ts

```diff
@@ -232,12 +239,23 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 	 * Construct an `EditLog` using the given options.
 	 * @param summary - An edit log summary used to populate the edit log.
 	 * @param logger - An optional logger to record telemetry/errors
+	 * @param editAddedHandlers - Optional handlers that are called when edits are added.
+	 * @param editLogSize - The target number of sequenced edits that the log will try to store in memory.
```

Comment:
I think this should be private actually. There's really no reason to ever use it over `earliestAvailableEditIndex`. I do agree it should be renamed anyways. `targetLength` sounds best to me since that's the wording I used in the comment.